proto/axelar/evm/v1beta1/events.proto

```diff
@@ -148,3 +148,11 @@ message CommandBatchAborted {
   bytes command_batch_id = 3 [ (gogoproto.customname) = "CommandBatchID" ];
   ;
 }
+
+message EVMEventFailed {
```

Comment:
```suggestion
message EventFailed {
```
Since the types are already namespaced under the module?

Reply:
I want to, there is already an EventFailed status